csharp/ql/lib/ext/System.model.yml

```diff
@@ -784,6 +784,36 @@ extensions:
+      - ["System", "UriBuilder", False, "UriBuilder", "(System.String,System.String,System.Int32,System.String,System.String)", "", "Argument[2]", "Argument[this].Property[System.UriBuilder.Port]", "taint", "manual"]
+      - ["System", "UriBuilder", False, "UriBuilder", "(System.String,System.String,System.Int32,System.String,System.String)", "", "Argument[3]", "Argument[this].Property[System.UriBuilder.Path]", "taint", "manual"]
+      - ["System", "UriBuilder", False, "UriBuilder", "(System.String,System.String,System.Int32,System.String,System.String)", "", "Argument[4]", "Argument[this].Property[System.UriBuilder.Query,System.UriBuilder.Fragment]", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Fragment", "()", "", "Argument[this].Property[System.UriBuilder.Fragment]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Host", "()", "", "Argument[this].Property[System.UriBuilder.Host]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Path", "()", "", "Argument[this].Property[System.UriBuilder.Path]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Port", "()", "", "Argument[this].Property[System.UriBuilder.Port]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Query", "()", "", "Argument[this].Property[System.UriBuilder.Query]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Scheme", "()", "", "Argument[this].Property[System.UriBuilder.Scheme]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "get_Uri", "()", "", "Argument[this].Property[System.UriBuilder.Scheme,System.UriBuilder.Host,System.UriBuilder.Port,System.UriBuilder.Path,System.UriBuilder.Query,System.UriBuilder.Fragment]", "ReturnValue", "taint", "manual"]
+      - ["System", "UriBuilder", False, "set_Fragment", "(System.String)", "", "Argument[0]", "Argument[this].Property[System.UriBuilder.Fragment]",  "taint", "manual"]
+      - ["System", "UriBuilder", False, "set_Host", "(System.String)", "", "Argument[0]", "Argument[this].Property[System.UriBuilder.Host]",  "taint", "manual"]
+      - ["System", "UriBuilder", False, "set_Path", "(System.String)", "", "Argument[0]", "Argument[this].Property[System.UriBuilder.Path]",  "taint", "manual"]
+      - ["System", "UriBuilder", False, "set_Port", "(System.Int32)", "", "Argument[0]", "Argument[this].Property[System.UriBuilder.Port]",  "taint", "manual"]
+      - ["System", "UriBuilder", False, "set_Query", "(System.String)", "", "Argument[0]", "Argument[this].Property[System.UriBuilder.Query]",  "taint", "manual"]
+      - ["System", "UriBuilder", False, "set_Scheme", "(System.String)", "", "Argument[0]", "Argument[this].Property[System.UriBuilder.Scheme]",  "taint", "manual"]
```

Comment:
There is no need to explicitly model these, unless you are worried about missing results in the `dotnet/runtime` repo?
When `System.UriBuilder` is used by other projects (and the source code for `UriBuilder` is not included) then it is assumed that *getters* and *setters of a property are reading/storing into the property.